src/rosetta-helpers.ts

```diff
@@ -103,11 +103,19 @@ type RosettaRevokeDelegateContractArgs = {
   result: string;
 };
 
-export function parseTransactionMemo(tx: BaseTx): string | null {
-  if (tx.token_transfer_memo && tx.token_transfer_memo != '') {
+export function parseTransactionMemo(memoHex: string | undefined): string | null {
```

Comment:
Note: this is currently bugged in the `develop` branch, it tries to trim NULL bytes off a hex string, which never does anything. Then, it returns the hex string without first decoding it to utf8 which is what the Rosetta response expects.

We may want to cherry pick this into develop before cutting a release off it.

Reply:
Done here: https://github.com/hirosystems/stacks-blockchain-api/pull/1292